components/automate-cli/cmd/chef-automate/infrastructure_test.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"context"
+	"testing"
+
+	api "github.com/chef/automate/api/interservice/deployment"
+	"github.com/chef/automate/lib/majorupgrade_utils"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/grpc"
+)
+
+var nodeDelCmd = &cobra.Command{
+	Use:   "node-delete [uuid]",
+	Short: "Delete node by node uuid",
+	Long:  "",
+	RunE:  runDeleteNodeCmd,
+	Args:  cobra.ExactArgs(1),
+}
+
+func Test_runpreInfrastructureCmd(t *testing.T) {
+	tests := []struct {
+		testName string
+		cmd      *cobra.Command
+		args     []string
+		wantErr  bool
+	}{
+		{"Test node delete", nodeDelCmd, []string{"uuid of node"}, true},
+	}
+	for _, tt := range tests {
+		t.Run(tt.testName, func(t *testing.T) {
+			if err := runDeleteNodeCmd(tt.cmd, tt.args); (err != nil) != tt.wantErr {
```

Comment:
condition is unnecessarily complex and the if condition also doesn't look correct, please simplify the conditions

Reply:
Done